### PR TITLE
1717 stanage going live move bulk of pages info into correct stanage tree

### PR DESCRIPTION
**Pull request by Carldkennedy**

### CODE REVIEW

**Comment by jkwmoore** [comment]
Closes https://github.com/rcgsheffield/sheffield_hpc/issues/1717

**Comment by Carldkennedy** [comment]
Rebase and move related files once these PRs are merged
https://github.com/rcgsheffield/sheffield_hpc/pull/1723
https://github.com/rcgsheffield/sheffield_hpc/pull/1704
https://github.com/rcgsheffield/sheffield_hpc/pull/1697

**Review by nikishe — APPROVED** [review]
LGTM .
Caught a small edit needed , but that can be ignored for now as I have a created a separate issue for it on #1730

File: stanage/cluster_specs.rst [hunk 1]
```diff
@@ -1,10 +1,10 @@
-.. _stanage-specs:
-
 .. toctree::
     :hidden:
     :maxdepth: 1
     :glob:
 
+.. _stanage-specs:
+
 Cluster Specifications
```
Comment (jkwmoore) [inline stanage/cluster_specs.rst:8]:
```suggestion
Stanage Specifications
```

This brings parity with the other top level cluster pages fyi.

File: stanage/slurm.rst [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.. _slurm_info_stanage:
+
+Slurm Scheduler Job Submission info
+===================================
+
+.. raw:: html
+
+    <meta http-equiv="refresh" content="0; URL=../hpc/scheduler/index.html#job-submission-control-on-stanage" />
+
+This page is now a stub, the details for job submission for Bessemer can now be found here: :ref:`Job Submission on Bessemer<stanage-job-submission>`
```
Comment (jkwmoore) [inline stanage/slurm.rst:10]:
```suggestion
This page is now a stub, the details for job submission for Stanage can now be found here: :ref:`Job Submission on Stanage <submit_job_stanage>`
```

I am fairly sure that this label is missing currently from https://github.com/rcgsheffield/sheffield_hpc/pull/1731/files#r1171094541

File: stanage/index.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
+.. _stanage:
+
+Stanage
+=======
+
+.. note::
+
+    The latest changes and newly installed software on our clusters can be found at the `HPC Changelog <http://changelog.hpc.shef.ac.uk/>`_ microsite.
```
Comment (jkwmoore) [inline stanage/index.rst:8]:
```suggestion
    The latest changes and newly installed software on our clusters can be found on the `HPC Changelog <http://changelog.hpc.shef.ac.uk/>`_ microsite.
```

**Comment by jkwmoore** [comment]
The current job failure is to be expected due to the use of a link label to the Stanage job submission info that currently exists in the other PR.

File: stanage/slurm.rst [hunk 4]
```diff
@@ -0,0 +1,10 @@
+.. _slurm_info_stanage:
+
+Slurm Scheduler Job Submission info
+===================================
+
+.. raw:: html
+
+    <meta http-equiv="refresh" content="0; URL=../hpc/scheduler/index.html#job-submission-control-on-stanage" />
+
+This page is now a stub, the details for job submission for Stanage can now be found here: :ref:`Job Submission on Stanage <submit_job_stanage>`
```
Comment (jkwmoore) [inline stanage/slurm.rst:10]: We will need to amend main to use this label then rebase. (Trying to follow the existing naming convention from the other clusters.)

File: stanagepilot/index.rst
Comment (jkwmoore) [inline stanagepilot/index.rst:1]: We mustn't forget to clean this bit up later too.

File: stanage/slurm.rst (hunk 4 above)
Comment (jkwmoore) [reply stanage/slurm.rst:10]: Rebased albeit looks a bit messy.

**Review by jkwmoore — COMMENTED** [review]
Late now, but yes approved :P

**Pull request merged**